spec/mailers/user_mailer_spec.rb

```diff
@@ -1306,6 +1325,19 @@ def expect_email_body_to_have_help_and_contact_links
       it_behaves_like 'a system email'
       it_behaves_like 'an email that respects user email locale preference'
 
+      it 'renders the idv_please_call template' do
+        user_mailer = UserMailer.with(user: user, email_address: email_address)
+
+        expect_any_instance_of(ActionMailer::Base).to receive(:mail).
+          with(hash_including(template_name: 'idv_please_call')).
+          and_call_original
+
+        user_mailer.in_person_please_call(
+          enrollment: enrollment,
+          visited_location_name: visited_location_name,
```

Comment:
Didn't we remove these?

Reply:
Yeah, but due to 50/50 state concerns I want to make sure this method doesn't barf if it gets these arguments passed to it

Reply:
(this test will go away in a subsequent PR)